### PR TITLE
COMMERCE-2401 Define default account role

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 buildscript {
 	dependencies {
-		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "latest.release"
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "6.1.77"
```
Comment (ethanbustad) [inline build.gradle]: We should remove this commit. This should be fixed by https://github.com/brianchandotcom/liferay-portal/pull/80545.

File: commerce-account-api/src/main/java/com/liferay/commerce/account/constants/CommerceAccountConstants.java [hunk 2]
```diff
@@ -23,6 +23,8 @@
 	public static final String ACCOUNT_ADMINISTRATOR_ROLE_NAME =
 		"Account Administrator";
 
+	public static final String ACCOUNT_BUYER_ROLE_NAME = "Buyer";
+
```
Comment (ethanbustad) [inline commerce-account-api/src/main/java/com/liferay/commerce/account/constants/CommerceAccountConstants.java]: All of these `*_ROLE_NAME` constants should probably be named `ROLE_NAME_*` instead. Ideally the specifics should come last -- and this actually goes for some of the other constants in this file. Examples of what I mean:

    ACCOUNT_BUYER_ROLE_NAME -> ROLE_NAME_ACCOUNT_BUYER
    ACCOUNT_GROUP_GUEST_NAME -> ACCOUNT_GROUP_NAME_GUEST
    ACCOUNT_ORDER_MANAGER_ROLE_NAME -> ROLE_NAME_ACCOUNT_ORDER_MANAGER
    DEFAULT_PARENT_ACCOUNT_ID -> ACCOUNT_ID_PARENT_DEFAULT (maybe?)
    GUEST_ACCOUNT_ID -> ACCOUNT_ID_GUEST

This type of naming makes it so similar constants are grouped together, which is very much desired in a class like this.

File: commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountUserRelLocalServiceImpl.java [hunk 3]
```diff
@@ -134,6 +144,49 @@ public void addCommerceAccountUserRels(
 		}
 	}
 
+	@Override
+	public void addDefaultRole(long userId, ServiceContext serviceContext)
+		throws PortalException {
+
+		User user = userPersistence.findByPrimaryKey(userId);
+
+		Set<Long> roleIdSet = new HashSet<>();
```
Comment (ethanbustad) [inline commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountUserRelLocalServiceImpl.java]: Why do we need this `Set`? It looks like it only ever has 1 entry, and it's only used to create an array, so we might as well just directly create an array of 1, yeah?

File: commerce-lang/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -77,6 +77,7 @@ always-ship-separately=Ship Separately
 an-email-has-been-sent-to-the-provided-email-address=An email has been sent to the provided email address.
 an-error-occurred-while-processing-your-payment=An error occurred while processing your payment.
 an-unexpected-error-occurred-while-invoking-the-geolocation-service-x=An unexpected error occurred while invoking the geolocation service: {0}.
+apply-default-role-to-existing-users=Apply Default Role To Existing Users
```
Comment (ethanbustad) [inline commerce-lang/src/main/resources/content/Language.properties]: value should be `Apply Default Role to Existing Users` -- lowercase `t` in `to`

**Review by ethanbustad — CHANGES_REQUESTED** [review]

**Comment by alessiorendina** [comment]
ci:test

**Comment by alessiorendina** [comment]
ci:test

**Comment by marco-leo** [comment]
@ethanbustad Merging, feel free to ask more changes

**Pull request merged**